### DISCUSSION TITLE
can I fasten the application review process

### DISCUSSION THREAD

**Discussion by manav019-spec** [community/community#146274] [GitHub Education]
### Hi everyone,

I have enrolled in the student developer pack as it was a task given to us and I have submitted my application for approval however as  we have due date for that task and my application is still not approved, I wanted approval as soon as possible or any status for that can it be possible??

**Comment by Eternatus9999** [comment]
NO, it takes time to review your application in order to lower the chance of major issues with your code the fast the review go the chance to miss code errors (e.g.: - showing sensitive data in console, Exception, etc.)

**Comment by davevad93** [comment]
Hi @manav019-spec .

GitHub no longer provide manual reviews, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @manav019-spec was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by manav019-spec** [comment]
oh! I see but is it possible in any case to get in contact with github management team by any chance for that .

**Reply by davevad93** [reply]
Hey @manav019-spec [GitHub Education support](https://support.github.com/contact/education) redirects here when selecting `Status of GitHub Education application (student)`

![0000](https://github.com/user-attachments/assets/b5f0d150-f076-4d0f-827c-576323ec9ac0)

So the only way is to wait, I hope you get it as soon as possible! 👍

**Comment by manav019-spec** [comment]
Yes...I hope so!!
